src/sdam/topology.ts

```diff
@@ -203,6 +203,8 @@ export class Topology extends TypedEventEmitter<TopologyEvents> {
   /** @internal */
   _type?: string;
 
+  client!: MongoClient;
```

Comment:
Users shouldn't access the client from the topology.  Should this property be marked as internal?

Reply:
The class is internal so there's no need, the surrounding internal documentation is unnecessary and we could remove it

Reply:
Yeah makes sense.  I was just thinking about consistency but I don't feel too strongly either way